tests/test_volume.py

```diff
@@ -314,11 +315,25 @@ def testFlip(self):
             self.assertTrue(isinstance(result, Volume))
 
     def testDownsample(self):
-        # Data files re-used from test_preprocess
         vols = Volume(np.load(os.path.join(DATA_DIR, "clean70SRibosome_vol.npy")))
-
-        resv = Volume(np.load(os.path.join(DATA_DIR, "clean70SRibosome_vol_down8.npy")))
-
         result = vols.downsample((8, 8, 8))
-        self.assertTrue(np.allclose(result, resv))
-        self.assertTrue(isinstance(result, Volume))
+        res = vols.resolution
+        ds_res = result.resolution
+
+        # check signal energy
+        self.assertTrue(
+            np.allclose(
+                anorm(vols.asnumpy(), axes=(1, 2, 3)) / res,
+                anorm(result.asnumpy(), axes=(1, 2, 3)) / ds_res,
+                atol=1e-3,
```

Comment:
Is `atol` related to the total energy in some way?  Just wondering why `1e-3`.

Reply:
Absolute tolerance level for `allclose`. The default is `1e-8`, but I had to bump up the tolerance for our downsample method to pass the test.

Reply:
If we use smooth Gaussian instead, we should get better energy conservation since we have much less energy in the high frequencies. It may make sense to do this (i.e., generate volumes using `gaussian_blobs` instead of loading this ribosome).